views/premium.html

```diff
@@ -1,157 +1,272 @@
-    <table class="widefat">
-      <thead>
-        <tr>
-         <td></td>
-         <td><strong><%= __("MailPoet") %></strong></td>
-         <td><strong><%= __("Web host") %></strong></td>
-         <td><strong><%= __("Third party") %></strong></td>
-        </tr>
-      </thead>
-      <tbody>
-        <tr class="alternate">
-         <td><strong><%= __("Speed") %></strong></td>
-         <td><%= __("50,000 / hour") %></td>
-         <td><%= __("300 / hour") %></td>
-         <td><%= __("2,000 / hour") %></td>
-        </tr>
-        <tr>
-         <td><strong><%= __("Daily email limits") %></strong></td>
-         <td><%= __("None") %></td>
-         <td><%= __("Yes") %></td>
-         <td><%= __("Depends") %></td>
-        </tr>
-        <tr class="alternate">
-         <td><strong><%= __("Personal deliverability support") %></strong></td>
-         <td><%= __("Yes!") %></td>
-         <td><%= __("No") %></td>
-         <td><%= __("No") %></td>
-        </tr>
-        <tr>
-         <td><strong><%= __("SPF and DKIM Signatures") %></strong></td>
-         <td><%= __("No need!") %></td>
-         <td><%= __("Update your DNS") %></td>
-         <td><%= __("Depends") %></td>
-        </tr>
-        <tr class="alternate">
-         <td><strong><%= __("Double opt-in") %></strong></td>
-         <td><%= __("Enforced") %></td>
-         <td><%= __("Not enforced") %></td>
-         <td><%= __("Depends") %></td>
-        </tr>
-        <tr>
-          <td colspan="4">
-            <div style="text-align: center">
-              <a
-                href="https://kb.mailpoet.com/article/181-comparison-table-of-sending-methods?utm_source=plugin&utm_medium=premium&utm_campaign=compare"
-                data-beacon-article="58a7182a2c7d3a576d3548c5"
-                target="_blank"
-              >
-                <%= __("View full comparison table") %>
-              </a>
-            </div>
-          </td>
-        </tr>
-      </tbody>
-    </table>
-    <p><%= __("Spammers are ineligible to use the MailPoet Sending Service. We reserve the right to cancel any sending plan if we detect more than 5% hard bounces. [link]Customers are required to clean their lists before joining MailPoet[/link].")
-      |replaceLinkTags('https://kb.mailpoet.com/article/127-checklist-before-importing-subscribers?utm_source=plugin&utm_medium=premium&utm_campaign=clean-lists', {'target': '_blank', 'data-beacon-article': '57ce07ffc6979108399a044b'})
-      |raw
-    %></p>
+  <div class="mailpoet-premium-page-text">
+    <p><%= _x('Try MailPoet Premium for 30 days with our', 'This text resides in the Premium page: /wp-admin/admin.php?page=mailpoet-premium') %></p>
```

Comment:
This should be closer to the header below, than border above. It needs special styling.